{{cookiecutter.project_name}}/pyproject.toml

```
@@ -8,6 +8,11 @@ readme = "README.rst"
 homepage = "https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}"
 repository = "https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}"
 documentation = "https://{{cookiecutter.project_name}}.readthedocs.io"
+{%- if cookiecutter.package_name != cookiecutter.project_name.replace('-', '_') -%}
```

```suggestion
{% if cookiecutter.package_name != cookiecutter.project_name.replace('-', '_') -%}
```

With `{%-` the `documentation` and `packages` keys run into each other.

See  https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control